include/global.h

```diff
@@ -264,7 +264,7 @@ extern double nav_y;
 extern double nav_z;
 
 extern double d; // Distanza generica.
-extern double k1; // Costanti ausiliarie.
+extern double k0, k1; // Costanti ausiliarie.
```

Comment:
These could probably just be local variables anyway, might be worth refactoring at some point.

Reply:
Yes, these global scratchpad variables are a nightmare. I only introduced k0 because I didn't feel comfortable using one that was already declared. Who knows if it would be used by some other routine.